src/patterns/data/default/cellPartOfAnatomicalEntity.tsv

```diff
@@ -1 +1,23 @@
-defined_class	defined_class_label	cell	anatomical_entity	pubs
+defined_class	defined_class_label	cell	anatomical_entity	pubs
+CL:6000011		CL:0000057	UBERON:0012399	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029	
+CL:6000012		CL:0000057	UBERON:0012398	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029
+CL:6000013		CL:0000057	UBERON:0007177	DOI:10.1111/j.1440-1827.1989.tb02421.x|DOI:10.1034/j.1600-065x.2003.00072.x|PMID:2618659|PMID:12969318
+CL:6000014		CL:0000165	UBERON:0007177	DOI:10.1111/j.1440-1827.1989.tb02421.x|DOI:10.1034/j.1600-065x.2003.00072.x|PMID:2618659|PMID:12969318
+CL:6000015		CL:0000165	UBERON:0011184	DOI:10.1159/000501521|PMID:31390635|PMCID:PMC6888884
+CL:6000016		CL:0000186	UBERON:0007177	DOI:10.1111/j.1440-1827.1989.tb02421.x|DOI:10.1034/j.1600-065x.2003.00072.x|PMID:2618659|PMID:12969318
+CL:6000018		CL:0000523	UBERON:0000088	DOI:10.1016/S0143-4004(05)80438-2|DOI:10.1016/S0143-4004(05)80438-2|DOI:10.1016/j.ajpath.2014.08.006|DOI:10.1016/j.jim.2003.03.001|DOI:10.1053/plac.2000.0559|DOI:10.1095/biolreprod.106.055244
+CL:6000020		CL:0002144	UBERON:0007177	DOI:10.1111/j.1440-1827.1989.tb02421.x|DOI:10.1034/j.1600-065x.2003.00072.x|PMID:2618659|PMID:12969318
+CL:6000021		CL:0002573	UBERON:0005304	PMID:2621148|PMCID:PMC1256735
+CL:6000022		CL:0002573	UBERON:0007177	DOI:10.1111/j.1440-1827.1989.tb02421.x|DOI:10.1034/j.1600-065x.2003.00072.x|PMID:2618659|PMID:12969318
+CL:6000023		CL:0002573	UBERON:0012398	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029
+CL:6000024		CL:0002573	UBERON:0012399	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029	
+CL:6000026		CL:0000510	UBERON:0011184	
+CL:6000027		CL:0009010	UBERON:0011184	
+CL:6000028		CL:0002204	UBERON:0011184	
+CL:6000029		CL:0000192	UBERON:0012398	
+CL:6000030		CL:0000192	UBERON:0012399	
+CL:6000031		CL:1000412	UBERON:0012399	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029
+CL:6000032		CL:1000412	UBERON:0012398	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029
+CL:6000033		CL:1000414	UBERON:0012399	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029
+CL:6000034		CL:1000414	UBERON:0012398	DOI:10.1007/978-1-4939-0897-4_3|PMID:24997029	
+CL:6000035		CL:0000115	UBERON:0003335	
```

Comment:
Values for "defined_class_label" are missing for all rows.
Values for "pubs" missing for several rows.

Reply:
I commented about the missing pubs here https://github.com/obophenotype/cell-ontology/pull/1722#issuecomment-1374109814 

The labels will be filled up with the DOS-DP. You need to check the `definions.owl` file to have a complete view.